src/ManageCourses.Api/Startup.cs

```diff
@@ -54,9 +54,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IDataService, DataService>();
 
             services.AddSingleton<IEmailService>(provider => new EmailService(
-                Configuration["email:host"],
-                Configuration["email:user"],
-                Configuration["email:password"]
+                Configuration["email:api_key"],
```

Comment:
Can you add token references here

https://github.com/DFE-Digital/manage-courses-api/blob/master/src/ManageCourses.Api/appsettings.SecretManager_Example.json

Can you also remove 
https://github.com/DFE-Digital/manage-courses-api/blob/396b7848229dcac4e5ebea979c031a2a94389eaf/src/ManageCourses.Api/appsettings.json#L15

Can we add the needed values here
https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/389349377/Manage+Courses+API

Reply:
Ah I keep forgetting where the examples live!! thanks :)